svg/elements/stop.json

```diff
@@ -172,9 +172,15 @@
               "safari": {
                 "version_added": "3"
               },
-              "safari_ios": {
-                "version_added": "3"
-              },
+              "safari_ios": [
```

Comment:
Looks great, @Havegum!

One suggestion I didn't notice until now: can we add `"mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Attribute/stop-opacity"` to this feature (just before the `"support"` key, outside the diff)